templates/organisations/_edit_form.html.twig

```diff
@@ -0,0 +1,27 @@
+{{ form_start(form) }}
+
+{{ form_row(form.name) }}
+{{ form_row(form.website) }}
+
+{% set addressPrototype = form_widget(form.contact.addresses.vars.prototype) %}
+
+{{ form_label(form.contact) }}
+
+{{ form_row(form.contact.firstName) }}
+{{ form_row(form.contact.lastName) }}
+{{ form_row(form.contact.email) }}
+{{ form_row(form.contact.phoneNumber) }}
+
+{{ form_label(form.contact.addresses) }}
+
+<div class="contact-addresses" data-prototype="{{ addressPrototype|e('html_attr') }}">
```

Comment:
can you add bootstrap classes to have a good display and improve user experience ?

Reply:
I'm using this class only for javascript, not styling